Equinox/Globals.h

```diff
@@ -54,5 +56,68 @@ typedef unsigned int uint;
 #define RESIZABLE true
 #define TITLE "Equinox Engine Editor"
 
+inline void DrawBoundingBox(AABB& boundingBox)
```

Comment:
Maybe this doesn't need a new module, could be put inside an existent and proper module. Nevertheless, when we create a new module, we don't need to create all that methods. And call the APP in the code when you just need a function, it's exactly the same that you're making with that global method. But, OK I'm just saying my opinion.